src/core/agent/services/keriaNotificationService.ts

```diff
@@ -347,6 +346,33 @@ class KeriaNotificationService extends AgentService {
     return true;
   }
 
+  private async processExnIpexOfferNotification(
+    notif: Notification
+  ): Promise<boolean>{
+    const exchange = await this.props.signifyClient.exchanges().get(notif.a.d);
```

Comment:
This can happen for ANY notification type. We need to fetch the exchange and check if we control the identifier of the sender of the inception event. If it does, we should skip.

If it doesn't, we should pass it as a parameter to each of the `process**` functions